packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/gux-form-field-radio.scss

```diff
@@ -126,3 +126,8 @@
   color: var(--gse-ui-radioButton-label-foregroundColor);
   vertical-align: middle;
 }
+
+.gux-input {
```

Comment:
Looks like this css change is causing the error gux-input element to shift down too much. 
<img width="200" alt="Screen Shot 2023-11-27 at 1 34 30 PM" src="https://github.com/MyPureCloud/genesys-spark/assets/127438502/886f5fd8-e319-4e32-b14d-a53448fe1f1e">

Reply:
This may be a more structural issue with how the components internal structure is composed.

Reply:
Hi Daragh, 

I just want to check. Do you mean by that I should look at restructuring the HTML rather than just changing the styling?

Reply:
Maybe, if you can't get the design to work or if restructuring the HTML would allow you to implement the design in a simpler way. The components have evolved over time. They started out with just the radio button/checkbox and label. Then an error message was added and then a help message was added. Basically if you think the css is getting overly complicated (or just not working) because of structural issues make a structural change.

Reply:
Ok, I went a different direction with the change. Kept it very light touch. No changes were required on the checkbox side